### DISCUSSION TITLE
Making a static site editable with Yellow?

### DISCUSSION THREAD

**Discussion by ty505** [datenstrom/community#597] [Ask a question] [answered]
How would I go about making a static site with an index.html and a css stylesheet function with Yellow?

**Comment by schulle4u** [comment, marked as answer]
Sorry, this is not possible at the moment. Yellow works as a static site generator. You can edit your pages in markdown using either your browser or a text editor in the file system and create a static HTML site, but not vice versa. To change your site you will have to edit your markdown content files, styles or layouts and then rebuild the static website from the [command line](https://github.com/datenstrom/yellow-extensions/tree/master/source/command).
